Rick-and-Morty/Rick And Morty/Views/CharacterDetailView.swift

```diff
@@ -17,10 +17,14 @@ struct CharacterDetailView: View {
                 .resizable()
                 .scaledToFit()
                 .frame(maxHeight: 200)
-            HStack {
+
+            VStack(alignment: .leading, spacing: 12) {
+                if let c = character as? ShortCharacterDescription {
+                    Text(c.shortDescription)
+                }
```

Comment:
This if this is a duplicated logic, IF there would be 3rd way of describing character, like `fullDescription` You would have fix this in 2 places in code. If the application would have more then those two, would you put that logic everywhere?